### ISSUE TITLE
Ignores `output-folder` when adding generated badges using git

### ISSUE THREAD

**Issue by calebpitan** [jpb06/jest-badges-action#46]
While my config specifies a custom output folder, it wasn't recognized when adding with git, the default ouput folder was still being used

**Error**

```bash
/usr/bin/git add ./badges
fatal: pathspec './badges' did not match any files
Error: 🔶 Oh no! An error occured: The process '/usr/bin/git' failed with exit code 128
```
**Config**

```yaml
 - name: Generating coverage badges
     uses: jpb06/jest-badges-action@latest
     with:
       output-folder: ./.badges
       coverage-summary-path: ./.coverage/coverage-summary.json
```

**Comment by calebpitan** [comment]
https://github.com/jpb06/jest-badges-action/blob/47d7718b8261d1549d9f2dedd67a868ecea6784d/src/logic/git/pushBadges.ts#L5

Is there a reason why this line doesn't use the `getInput('output-folder')` call?

**Comment by calebpitan** [comment]
I already left a pull request for this https://github.com/jpb06/jest-badges-action/pull/47

**Comment by jpb06** [comment]
Hello!

Oof! Good catch! Sorry, I missed that one.

Thank you!

**Comment by jpb06** [comment]
Released as [1.9.10](https://github.com/jpb06/jest-badges-action/releases/tag/v1.9.10) 🙇🏻

**Comment by jpb06** [comment]
Closing this. Let me know if anything is missing 🙇🏻
